minidump-processor/json-schema.md

```diff
@@ -171,7 +171,9 @@ anyway.
            "nearby_registers": <u32>
         },
         /// The calculated confidence value in the bit-flip-corrected address.
-        "confidence": <f32>
+        "confidence": <f32>,
+        /// The source register from which the address was retrieved, if from a register at all.
```

Comment:
Does this mean that it's a string if it was retrieved from a register and null if not?

Reply:
Yes. Would you prefer it be omitted entirely if unset?

Reply:
I don't have a preference for the behavior, but it helps to have a clear schema so that I know what's in the output, how it's shaped, and how to process it.